Pod/Classes/DateConvenience.swift

```diff
@@ -0,0 +1,71 @@
+//
+//  DateConvenience.swift
+//  Pods
+//
+//  Created by Jason Grandelli on 11/14/16.
+//
+//
+
+import Foundation
+
+public extension Date {
+    // MARK: Units
+    public var components: DateComponents? {
+        return Calendar.current.dateComponents([.era, .year, .month, .day, .hour, .minute, .second, .weekday, .weekdayOrdinal, .quarter, .weekOfMonth, .weekOfYear, .yearForWeekOfYear, .nanosecond, .calendar, .timeZone], from: self)
+    }
+
+    public static func monthSymbols() -> [String] {
+        return DateFormatter().monthSymbols
+    }
+    
+    public static func monthSymbol(month: Int) -> String? {
```

Comment:
we _could_ but optional here works fine i guess. kinda. the pattern for all these is optionals. we could evaluate where to use throwing